ipynb/tests/Frequency Invariance Test.ipynb

```diff
@@ -0,0 +1,791 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup"
```

Comment:
The notebook starts a bit abrupt. It would be best to start with:
# Frequency Invariant Test

"FreqInvarianceTest is a LisaTest class for automated testing of frequency invariant load tracking. This notebook uses the methods it provides to perform the same analysis as the automated test and plot some results."

And maybe add here:
"Target setup is done based on the information in target.config."

## Setup
[snip]